ref/src/sw_driver.F90

```diff
@@ -166,14 +225,22 @@ program sw_driver
 
   ! Turn off GPTL if enabled
 #ifdef ENABLE_GPTL
-  if (do_profile == 1) then
-    ret = gptlpr(0)
-    ret = gptlfinalize()
+  if (do_profile == 1 .or. mpi_run) then
```

Comment:
Why do we need `mpi_run` here?  I thought we were getting rid of the need for that variable.

Reply:
Yes, that is what I had in mind.  We should only see GPTL output if `do_profile=1`.